packages/terra-worklist-data-grid/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import '../_elementPolyfill';
+import * as KeyCode from 'keycode-js';
+import styles from '../WorklistDataGrid.module.scss';
+import WorklistDataGridUtils from '../utils/WorklistDataGridUtils';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * String identifier of the column in which the Cell will be rendered.
+   */
+  columnId: PropTypes.string.isRequired,
+  coordinates: PropTypes.shape({
+    x: PropTypes.number,
+    y: PropTypes.number,
+  }),
+  /**
+   * Boolean value to indicate if the cell can accept focus.
+   */
+  acceptsFocus: PropTypes.bool,
+  width: PropTypes.string,
+  height: PropTypes.string,
+  onMoveCellFocus: PropTypes.func,
+  onCellSelectionChange: PropTypes.func,
+  /**
+   * Content that will rendered within the Cell.
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  acceptsFocus: false,
+};
```

Comment:
Do you really need this default prop?  If this prop is not passed, it will already evaluate as a falsey value.

Reply:
This is one of the conventions in Terra so I would say to keep it for consistency.